akka-diagnostics/src/main/resources/reference.conf

```diff
@@ -9,6 +9,214 @@
+    akka.cluster.singleton.hand-over-retry-interval,
+    akka.cluster.singleton.min-number-of-hand-over-retries,
+    akka.cluster.singleton-proxy.singleton-name,
+    akka.cluster.singleton-proxy.singleton-identification-interval,
+
+    # akka-cluster-sharding
+    akka.cluster.sharding.guardian-name,
+    akka.cluster.sharding.retry-interval,
+    akka.cluster.sharding.shard-failure-backoff,
+    akka.cluster.sharding.entity-restart-backoff,
+    akka.cluster.sharding.rebalance-interval,
+    "akka.cluster.sharding.coordinator-singleton.*",
+
+    # akka-persistence
+    akka.persistence.no-snapshot-store.class,
+    "akka.persistence.dispatchers.*",
+    "akka.persistence.journal-plugin-fallback.*",
+    "akka.persistence.snapshot-store-plugin-fallback.*"
+    "akka.persistence.journal.inmem.*",
+    akka.persistence.snapshot-store.local.class,
+    akka.persistence.snapshot-store.local.plugin-dispatcher,
+    akka.persistence.journal.leveldb.class,
+    akka.persistence.journal.leveldb.plugin-dispatcher,
+    akka.persistence.journal.leveldb.fsync
+  ]
+
+  # Definition of sections or properties that are not checked
+  # for typos and misplacements, because they are supposed to
+  # be used. You should use 'confirmed-typos' instead of modifying
+  # this list.
+  disabled-typo-sections = [
+    "akka.actor.serializers",
+    "akka.actor.serialization-bindings",
+    "akka.actor.serialization-identifiers",
+    "akka.actor.mailbox.requirements",
+    "akka.persistence.publish-plugin-commands",
+    "akka.coordinated-shutdown.phases",
+    "akka.coordinated-shutdown.reason-overrides",
+    "akka.cluster.role",
+    # kryo mappings are classnames of what one wants to use kryo for, so they won't be classes not known to us, but good.
+    "akka.actor.kryo.mappings",
+    # Artery is still experimental, so let's not warn about typos here yet
+    "akka.remote.artery",
+    # These are special, empty-by-default settings in Akka HTTP
+    "akka.http.server.parsing", # empty by default, but used to override values from akka.http.parsing
+    "akka.http.client.parsing", # empty by default, but used to override values from akka.http.parsing
+    "akka.http.host-connection-pool.parsing", # empty by default, but used to override values from akka.http.parsing
+    "akka.http.host-connection-pool.client" # empty by default, but used to override values from akka.http.client,
+    "akka.persistence.multi-data-center.cross-reading-replication.cassandra-journal",
+    # akka-discovery
+    "akka.discovery.config.services",
+    # akka-management
+    "akka.management.http.routes",
+    "akka.management.health-checks.readiness-checks",
+    "akka.management.health-checks.liveness-checks",
+    # Not covered by config checking
+    "akka.kafka",
+    # Even if we add akka-projection-slick and jackson serializer, most are dynamic so easier to add it here
+    "akka.projection.slick",
+    "akka.serialization.jackson",
+  ]
+}
+#//#config-checker
```

Comment:
this `#//#config-checker` must be removed otherwise `sbt docs/paradox` will fail
we don't want to show that section in docs

Reply:
I will delete the one in line 55, so we have `//#config-checker` between lines 14 and 217

Reply:
removed `includes/common.md`